recipes/google-cloud-webrisk/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "google-cloud-webrisk" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "1dee50964303c4e2b9e13b9ddd7ff72a80be3dac04c725e9d051692d715b37a3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python
+    - pip >=18.1
+    - google-api-core[grpc] >=1.14.0,<2.0.0dev
```

Comment:
I don't think conda supports the extras syntax. I think you would need to just add whatever `grpc` pulls in as separate dependencies. Also is the `google-api-core` required to build the package? If it is only a runtime dependency, just list it under `run`.